src/resolver.ts

```diff
@@ -1,15 +1,30 @@
 import { UserInput } from './UserInput';
 import { User } from './entity/User';
 import { AppDataSource } from './data-source';
+import { CustomError } from './custom-errror';
+import { Md5 } from 'ts-md5';
 
 export const resolvers = {
   Mutation: {
     createUser: async (_, args: { data: UserInput }) => {
       const { data } = args;
+      if (data.password.length < 6) {
+        throw new CustomError('A senha deve conter pelo menos 6 caracteres', 400);
+      }
+      if (!thereIsLetter(data.password)) {
```

Comment:
```suggestion
      if (!isThereALetter(data.password)) {
```

Funções e constantes booleanas no geral geralmente é legal nomear como se fosse uma pergunta. Assim fica mais claro né?